avalon/tools/contextmanager/__init__.py

```diff
@@ -1,3 +1,3 @@
-from app import show
+from .app import show
```

Comment:
Oh wow, this was not already the case? Surprised this hasn't broken in some way already.

Reply:
Well spotted, and thanks. I'm quite surprised too. But 🥇!